### PR TITLE
add shebang to rc.local snippets

### PR DESCRIPTION
**Pull request by dunn**
Usually should be obvious, but if you're creating `rc.local` from scratch this is a good reminder that the shebang is required.

### CODE REVIEW

File: source/deploy/deploy/_standalone.md.erb [hunk 1]
```diff
@@ -103,6 +103,7 @@ Here is an example of what you may want to add to `/etc/rc.local`. If there is a
 <span class="nb">export</span> <span class="nv">PATH</span>=<span class="o">/path-to-passenger</span>/bin:<span class="nv">$PATH</span>
 
 <% end -%>
+<span class="c">#!/bin/sh</span>
```
Comment (FooBarWidget) [inline source/deploy/deploy/_standalone.md.erb]: I think the shebang line doesn't belong here, but earlier in the block?

File: source/walkthroughs/deploy/deploy_app/_standalone_instructions.md.erb [hunk 2]
```diff
@@ -152,6 +152,7 @@ Here is an example of what you may want to add to `/etc/rc.local`. If there is a
 <span class="nb">export</span> <span class="nv">PATH</span>=<span class="o">/path-to-passenger</span>/bin:<span class="nv">$PATH</span>
 
 <% end -%>
+<span class="c">#!/bin/sh</span>
```
Comment (FooBarWidget) [inline source/walkthroughs/deploy/deploy_app/_standalone_instructions.md.erb]: I think the shebang line doesn't belong here, but earlier in the block?

**Comment by dunn** [comment]
Placement of those two shebangs fixed, thanks!

**Comment by FooBarWidget** [comment]
Thanks!

**Pull request merged**